js/modules/k6/k6.go

```diff
@@ -182,6 +184,13 @@ func (*K6) Check(ctx context.Context, arg0, checks goja.Value, extras ...goja.Va
 				succ = false
 			}
 		}
+
+		// Fail if a single iteration caused an error
```

Comment:
`iteration` is not the correct term here, it might be confused with a script iteration - `check callback` is probably a better term.

Reply:
I removed this comment as well, it's self-explanatory really.